rudder_airflow_provider/hooks/rudderstack.py

```diff
@@ -62,6 +61,13 @@ def poll_for_status(self):
                 break
             time.sleep(STATUS_POLL_INTERVAL)
 
+    def get_request_headers(self) -> dict:
+        access_token = self.get_access_token()
+        return {
+            'authorization': f"Bearer {access_token}",
```

Comment:
Can you help me understand what `f` means in `f"Bearer {access_token}"`?

Reply:
f allows embedding expressions in strings. `access_token` is the variable and replaced with value.